lib/esc_pos_utils/src/generator.dart

```diff
@@ -122,16 +122,19 @@ class Generator {
     switch (styles.fontSize) {
       case Size.small:
         if (_styles.fontSize != Size.small) {
+          bytes += _resetLinesSpacingCommand();
+          bytes += _lettersSpacingCommand(Size.small.letterSpacing);
           bytes += cFontA.codeUnits;
           bytes += Uint8List.fromList(
             List.from(cSizeGSn.codeUnits)..add(PosTextSize.decSize(styles.fontSize)),
           );
-          _styles = _styles.copyWith(fontSize: styles.fontSize);
+          _styles = _styles.copyWith(fontSize: Size.small);
         }
         break;
       case Size.large:
         if (_styles.fontSize != Size.large) {
           bytes += _linesSpacingCommand();
```

Comment:
Why at small we have `_resetLinesSpacingCommand` and at large `_linesSpacingCommand`?